gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/ConnectionPool.java

```diff
@@ -369,15 +345,6 @@ private boolean addConnectionIfUnderMaximum() {
     }
 
     private boolean destroyConnection(final Connection connection) {
```

Comment:
Nit: this and definitelyDestroyConnection() could get merged into single method

Reply:
Merged